site/_variables.yml

```
@@ -1,19 +1,44 @@
-# Product names
-vm_platform: "ValidMind Platform UI"
-vm_framework: "ValidMind Developer Framework"
-vm_ui: "platform UI"
-vm_dev: "developer framework"
-vm_risk: "ValidMind AI risk platform"
-
-# Product URLs
+# VARIABLE NAMING CONVENTIONS
+# - Variables for long & short forms should be related (e.g. `developer_framework`, `developer`)
+# - Variable names should be descriptive and consistent (e.g. `user_first_name`, `user_last_name`)
+# - Use prefixes / suffixes for grouping (e.g. `orderDate`, `orderAmount`)
+# - Use snake case (e.g. `developer_framework`)
+# - Avoid abbreviations (e.g. `customer_address` NOT `custAddr`) — `vm` for ValidMind is one exception
+# - Include enough contextual information (e.g. `invoiceTotal` NOT `total`)
+
+
+
+# PRODUCT NAMES
+# Names of ValidMind products or core features
+
+# Long form
+validmind_platform: "ValidMind Platform UI"
+validmind_developer: "ValidMind Developer Framework"
+risk_platform: "ValidMind AI risk platform"
+
+# Short form
+platform: "platform UI"
```

```suggestion
platform: "platform UI"
```

I like this! In the past, I've had to use `_short` and `_long` to differentiate between long form and short form variables but this is actually much more elegant. Your short form literally mirrors how the English shortform is constructed, by dropping the "ValidMind" moniker.